webapp/components/compat-2021.js

```diff
@@ -6,10 +6,152 @@
 
 import {load} from '../node_modules/@google-web-components/google-chart/google-chart-loader.js';
 import '../node_modules/@polymer/paper-button/paper-button.js';
+import '../node_modules/@polymer/paper-dialog/paper-dialog.js';
+import '../node_modules/@polymer/paper-input/paper-input.js';
 import '../node_modules/@polymer/polymer/lib/elements/dom-if.js';
 import { html, PolymerElement } from '../node_modules/@polymer/polymer/polymer-element.js';
 
 const GITHUB_URL_PREFIX = 'https://raw.githubusercontent.com/Ecosystem-Infra/wpt-results-analysis/gh-pages';
+const SUMMARY_FEATURE_NAME = 'summary';
+const FEATURES = [
+  'aspect-ratio',
+  'css-flexbox',
+  'css-grid',
+  'css-transforms',
+  'position-sticky',
+];
+
+// Compat2021DataManager encapsulates the loading of the CSV data that backs
+// both the summary scores and graphs shown on the Compat 2021 dashboard. It
+// fetches the CSV data, processes it into sets of datatables, and then caches
+// those tables for later use by the dashboard.
+class Compat2021DataManager {
+  constructor() {
+    this._dataLoaded = load().then(() => {
+      return Promise.all([this._loadCsv(true), this._loadCsv(false)]);
+    });
+  }
+
+  // Fetches the datatable for the given feature and stable/experimental state.
+  // This will wait as needed for the underlying CSV data to be loaded and
+  // processed before returning the datatable.
+  async getDatatable(feature, stable) {
```

Comment:
```suggestion
  async getDataTable(feature, stable) {
```

Since these are `google.visualization.DataTable`, and also with the "atat" in this word it's easier to read like this.